Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:jessie
+MAINTAINER PotHix
+RUN apt-get update
```

Comment:
Only one issue here. Running `apt-get update` alone in a statement causes caching issues. It think we'll be better off with something like this:

```
RUN apt-get update && apt-get install -y --force-yes /
```